idl/github.com/uber/cadence/shared.thrift

```diff
@@ -109,6 +110,9 @@ enum EventType {
   RequestCancelExternalWorkflowExecutionFailed,
   ExternalWorkflowExecutionCancelRequested,
   MarkerRecorded,
+  SignalExternalWorkflowExecutionInitiated,
+  SignalExternalWorkflowExecutionFailed,
+  ExternalWorkflowExecutionSignalRequested,
```

Comment:
This name follows CancelExternalWorkflow, but should I change it to something like "SignalExternalWorkflowExecutionSuccess"?

Reply:
Ok, I got answer from sammar that this name is consistent with SWF, so will keep it like this.